core/src/main/java/io/github/tcdl/msb/support/Utils.java

```diff
@@ -56,8 +58,8 @@ public static String toJson(Object object, ObjectMapper objectMapper) {
         try {
             return objectMapper.writeValueAsString(object);
         } catch (JsonProcessingException e) {
-            LOG.error("Failed parse to JSON from: " + object, e);
-            throw new JsonConversionException(e.getMessage());
+            LOG.error("Failed to parse to JSON object: [{}] ", object);
```

Comment:
I would say i improved a lot. We log only in Utils ->
Failed to parse from JSON: [{}] to object of type:
Failed to parse to JSON object: [{}]
Failed to convert object [{}] to type:
So Utils logs context it knows about -> objects, raw json and most important - type we convert into.

Actual callers to this are Producer/Consumer/DefaultChannelMonitorAggregator (but this is runtime so we not insist on this as result can't expect they will log or smth) just catch JsonConversionException and log exception with context they know. 

Example in DefaultChannelMonitorAggregator log:

ERROR [timer-provider-thread-1] Utils Failed to convert object [{"body":" unexpected statistics format received"}] to type: [io.github.tcdl.msb.api.message.payload.Payload<java.lang.Object, java.lang.Object, java.lang.Object, java.util.Map<java.lang.String, io.github.tcdl.msb.monitor.agent.AgentTopicStats>>]

12:35:31,362 ERROR [timer-provider-thread-1] DefaultChannelMonitorAggregator Unable to convert message.
io.github.tcdl.msb.api.exception.JsonConversionException: Can not instantiate value of type java.util.LinkedHashMap from String value (' unexpected statistics format received'); no single-String constructor/factory method
 at [Source: N/A; line: -1, column: -1]
	at io.github.tcdl.msb.support.Utils.convert(Utils.java:107)

Example in ResponderServerImpl:
12:41:07,292 ERROR [main] Utils Failed to convert object [{"body":"some body"}] to type: [io.github.tcdl.msb.api.message.payload.Payload<?, ?, ?, java.lang.Integer>]
12:41:07,292 ERROR [main] ResponderServerImpl [test:responder-server] Error while processing message with id: [090c2bfa-7ca5-4457-bde2-c23fd7e639ef]. Cause: [Can not construct instance of int from String value 'some body': not a valid Integer value
 at [Source: N/A; line: -1, column: -1]]